src/components/FilterPanel/FilterPanelCheckboxWithOverflowMenu/FilterPanel.stories.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file Filter panel checkbox with overflow menu stories.
+ * @copyright IBM Security 2020
+ */
+
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { text, number } from '@storybook/addon-knobs';
+
+import React from 'react';
+
+import { patterns } from '../../../../.storybook';
+
+import FilterPanelCheckboxWithOverflowMenu from '.';
+import FilterPanelSearch from '../FilterPanelSearch';
+import OverflowMenuItem from '../../OverflowMenuItem';
```

Comment:
As an extra measure for verifying the public API, we've been importing components for stories directly from `src/index.js` - For example, in this case it might be `import { FilterPanelCheckboxWithOverflowMenu, FilterPanelSearch, OverflowMenuItem } from '../../..'`